README.md

```diff
@@ -13,66 +13,113 @@ Assumptions:
 * You have git installed on your machine and are able to clone code repositories from GitHub. If this isn't the case, the easiest way to get started is to install [GitHub Desktop](https://desktop.github.com/ "GitHub desktop"), available for Windows or Mac.
 * You're familiar with opening a terminal on your machine and using the command line as needed.
 
-### Install Prerequisites and Code
+### Installation
+_Get the code and install the runtime and dependencies_
 
 1. If you're not already running Node.js, download and run the installer for your operating system. We build using **Node.js 10.15.3 (LTS)** which we recommend downloading using [Node Version Manager (nvm)](https://github.com/nvm-sh/nvm).
```

Comment:
we're actually now using `12.18.4`

Reply:
I didn't change the version docs but can to be consistent, and make sure it matches node version used the dockerfile.